packages/tools/devtools/devtools/api-report/devtools.api.md

```diff
@@ -32,7 +31,7 @@ export interface IDevtools extends IDisposable {
     registerContainerDevtools(props: ContainerDevtoolsProps): void;
 }
 
-// @public
+// @public (undocumented)
```

Comment:
Note: this is misleading. It actually is documented, just via `inheritDoc`. API-Extractor doesn't handle this case properly :(